docs/release-notes/january-2024.md

```diff
@@ -0,0 +1,111 @@
+---
+slug: january-2024
+hide_title: true
+description: >-
+    Release notes include the changes, fixes, and additions in specific versions of Semgrep.
+toc_max_heading_level: 3
+title: January 2024
+tags:
+  - Release notes
+---
+
+# Semgrep release notes for January 2024
+
+## 🔧 OSS Engine
+
+* The following versions of the OSS Engine were released in January 2024:
+  * [<i class="fas fa-external-link fa-xs"></i>1.55.1](https://github.com/semgrep/semgrep/releases/tag/v1.55.1)
+  * [<i class="fas fa-external-link fa-xs"></i>1.55.2](https://github.com/semgrep/semgrep/releases/tag/v1.55.2)
+  * [<i class="fas fa-external-link fa-xs"></i>1.56.0](https://github.com/semgrep/semgrep/releases/tag/v1.56.0)
+  * [<i class="fas fa-external-link fa-xs"></i>1.57.0](https://github.com/semgrep/semgrep/releases/tag/v1.57.0)
+  * [<i class="fas fa-external-link fa-xs"></i>1.58.0](https://github.com/semgrep/semgrep/releases/tag/v1.58.0)
+  * [<i class="fas fa-external-link fa-xs"></i>1.59.0](https://github.com/semgrep/semgrep/releases/tag/v1.59.0)
+
+## 🌐 Cloud Platform
+
+### Added
+
+* Semgrep's Visual Studio Code extension now runs natively on Windows machines.
+* Added ability for organizations to test connections to source code managers by going to
+  **Settings** > **Source Code Managers**.
+* Added ability to manually create custom dependency exceptions under **Supply
+  Chain** > **Settings**. This helps prevent blocking a pull request or merge
+  request due to licensing issues. For example, if `bitwarden/cli@2023.9.0`,
+  which has a GPL-3.0 license, is on the allowlist, setting a custom dependency
+  exception means that the exclusion won't fail when upgrading to
+  `bitwarden/cli@2023.9.1`.
+* Projects are removed from SCP when the corresponding GitHub repository is
+  archived on GitHub.
+* **CLI tool**: 
+  * Added color-coded severity icons, such as `❯❯❱`, to the CLI
+  output for findings of known severity.
+  * Metrics sent from the CLI to Semgrep now include a breakdown of the number
```

Comment:
Do you know which team added the metric in? I just checked our `privacy.md` and we don't mention it at all: https://github.com/semgrep/semgrep/blob/develop/PRIVACY.md#data-collected-as-metrics - I can follow up with them to have it added in.

Reply:
Yup, it's https://linear.app/semgrep/issue/PA-3312/add-findings-per-product-to-cli-scan-metrics (PM and/or Growth?)